codec-http2/src/main/resources/META-INF/native-image/io.netty/codec-http2/native-image.properties

```diff
@@ -12,5 +12,5 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-Args = --rerun-class-initialization-at-runtime=io.netty.handler.codec.http2.Http2CodecUtil \
```

Comment:
@pmlopes why did you use `--rerun-class-initialization` here? I could not find any substitutions for this where this might be necessary.

Reply:
Answer here: https://github.com/netty/netty/pull/8963#discussion_r280351904 with old combinations of netty / graalvm this was needed, with newer ones `rc16` inclusive this isn't needed anymore and can be just as you proposed.